tests/unit/records/test_views.py

```diff
@@ -296,3 +296,22 @@ def test_stats(app, db, minimal_record, stats, expected_result):
         assert expected_result['unique_views'] in template
         assert expected_result['unique_downloads'] in template
         assert 'All versions' not in template
+
+
+def test_record_thumbnail(app, record_with_image_creation):
+    pid, record, record_url = record_with_image_creation
+    for precached_thumbnail in current_app.config['PRECACHED_THUMBNAILS']:
+        with app.test_client() as client:
+            res = client.get(record_url + '/thumbnail/' + precached_thumbnail)
```

Comment:
Since it's a proper view we can use `flask.url_for` to get the proper URL to use with the test client